enterprise/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/compatibility/v3_4/runtime/compiled/codegen/ir/LoopDataGenerator.scala

```diff
@@ -33,5 +33,7 @@ trait LoopDataGenerator {
 
   def produceLoopData[E](iterVarName: String, generator: MethodStructure[E])(implicit context: CodeGenContext): Unit
 
+  def close[E](iterVarName: String, generator: MethodStructure[E]): Unit
```

Comment:
We could do `iterVarName` -> `cursorName` around here too!

Reply:
`LoopDataGenerator` is used both for iterators and cursors, hence the confusion